rclpy/rclpy/node.py

```diff
@@ -27,14 +27,16 @@ def __init__(self, handle):
     def create_publisher(self, msg_type, topic, qos_profile):
```

Comment:
Confirmed that this parameter is optional in rclcpp API so I'll make it optional here as you suggested. Thanks for pointing it out.
I don't know if we'll simplify the examples though, underlying question is "do we want to show that the API allows you to pass qos profiles from the highest level ?"